pkg/shell/plot.js

```diff
@@ -130,12 +130,16 @@ shell.plot = function plot(element, x_range_seconds, x_stop_seconds) {
     var options = { };
     var data = [ ];
     var flot = null;
-    var walk_timer;
     var result = { };
 
-    var now, x_range, x_offset, interval;
+    var interval;
+
+    var grid;
 
     function refresh() {
+        if (!grid)
```

Comment:
Why would there ever be no grid? Shouldn't a graph always have a grid? Although the grid may have no rows (yet), and those rows may have no data (yet) ... it should always have a grid. The grid is the representation of the bounds of the graph.